### DISCUSSION TITLE
How do you install this?

### DISCUSSION THREAD

**Discussion by lavastone177** [Trollhunters501/Barrel#3] [Q&A] [answered]
Can you specify which zip it is? I could not find it. Sorry!

**Comment by Trollhunters501** [comment, marked as answer]
> Can you specify which zip it is? I could not find it. Sorry!

In releases there is a jar file, execute it
If you use the nukkit plugin, put it in the corresponding folder

**Reply by lavastone177** [reply]
Thank you!! Appreciate the help. Nice plugin though.
